benchmark/inference/inference_benchmark.py

```diff
@@ -108,26 +109,28 @@ def run(args: argparse.ArgumentParser) -> None:
                             metadata=data.metadata() if hetero else None)
                         model = model.to(device)
                         model.eval()
-
-                        with amp:
-                            for _ in range(args.warmup):
-                                model.inference(subgraph_loader, device,
-                                                progress_bar=True)
-                            with timeit():
-                                # becomes a no-op if vtune_profile == False
-                                with emit_itt(args.vtune_profile):
+                        with subgraph_loader.enable_cpu_affinity(
```

Comment:
There is a lot of nesting going on here, but it looks like Python supports chaining of context managers. For example, we could simplify this to:
```python
if args.cpu_affinity:
    cpu_affinity = subgraph_loader.enable_cpu_affinity(args.loader_cores)
else:
    cpu_affinity = nullcontext()
profile = torch_profile() if args.profile else nullcontext()
with cpu_affinity, amp, timeit(), emit_itt(args.vtune_profile):
    pass
```

Reply:
Changed for cleaner structure as suggested. I also added `reset()` function to the `timeit()` module, because I found it useful to measure times for warmup and inference loops. With reset all can be defined in the same line.